server/utils.ts

```diff
@@ -0,0 +1,5 @@
+export const isString = (obj: any): obj is string =>
+  Object.prototype.toString.call(obj) === "[object String]";
+
+export const isObject = (obj: any): obj is Object =>
+  obj !== null && typeof obj === "object";
```

Comment:
https://lodash.com/docs/#isPlainObject

Reply:
This is a typeguard have some more feature then is plain object.
Do you mean replace whole thing with lodash or just a body ?

Reply:
I think to start implement it in this project. This lib should be helpful for some array, string, object variables

Reply:
This function is not the same from technical point of view. This is typescript type guard
`obj is Object`